olive/evaluator/metric.py

```diff
@@ -79,6 +79,7 @@ def validate_goal(cls, v, values):
 class Metric(ConfigBase):
     name: str
     type: MetricType
+    backend: Optional[str] = "torch_metrics"
```

Comment:
>Optional [](http://example.com/codeflow?start=13&length=8)

could the backend be None?

Reply:
Yes, for customized metric, it should be None.